plugins/ci/cmd/insights-ci/main.go

```diff
@@ -15,19 +15,11 @@ import (
 
 const maxLinesForPrint = 8
 
-func exitWithError(message string, err error) {
-	if err != nil {
-		logrus.Fatalf("%s: %s", message, err.Error())
-	} else {
-		logrus.Fatal(message)
-	}
-}
-
 func main() {
 	ciScan, err := ci.NewCIScan()
-	defer ciScan.Close()
+	defer ciScan.Close() // TODO: when exitWithError, close is not executed (refactor)
```

Comment:
it looks like all the `exitWithError` calls are in main, so maybe it could just take `ciScan` as an argument? I don't think it's a big deal to leave the temp files though since it's in a container.

Reply:
Agreed, this is mainly to improve local testing flow